### ISSUE TITLE
[BUG] Blank PDF on AWS Lambda 

### ISSUE THREAD

**Issue by katunch** [alixaxel/chrome-aws-lambda#170]
## Environment
* `chrome-aws-lambda` Version: `^5.3.1`
* `puppeteer` / `puppeteer-core` Version: `^5.3.1`
* OS: macOS
* Node.js Version: `12.x`
* Lambda / GCF Runtime: `nodejs12.x`

## Expected Behavior

The html on page should be exported into the pdf

## Current Behavior

The generated pdf is blank.

## Steps to Reproduce
I have a repository which the whole code is stored: https://github.com/softwarebrauerei/pdf-renderer
```js
'use strict'

const chromium = require('chrome-aws-lambda');
const fs = require('fs');
const path = require('path');
const Mustache = require('mustache');

let response;

module.exports.render = async (event, context) => {
    const body = JSON.parse(event.body);
    if (!(body.template === null && body.data === null)) {
        const template = body.template;
        const templateData = body.data;

        const renderedTemplateHtml = Mustache.render(template, templateData);
        const fileName = `pdf.pdf`;

        let browser = null;
        try {
            browser = await chromium.puppeteer.launch({
                args: chromium.args,
                defaultViewport: chromium.defaultViewport,
                executablePath: await chromium.executablePath,
                headless: chromium.headless,
                dumpio: false,
                ignoreHTTPSErrors: true,
            });

            const page = await browser.newPage();
            await page.setContent(renderedTemplateHtml, {
                waitUntil: ['networkidle0','load','domcontentloaded'],
                timeout: 30000
            })

            const pdf = await page.pdf({
                format: 'A4',
                printBackground: true,
                margin: {
                    top: '1cm',
                    right: '1cm',
                    bottom: '1cm',
                    left: '1cm'
                }
            });

            response = {
                headers: {
                    'Content-Type': 'application/pdf',
                    // 'content-disposition': `attachment; filename=${fileName}`,
                    'Content-Length': Buffer.byteLength(pdf, 'utf-8')
                },
                statusCode: 200,
                body: pdf.toString('base64'),
                isBase64Encoded: true
            };
        }
        catch (error) {
            response = {
                statusCode: 500,
                body: JSON.stringify({
                    statusCode: 500,
                    message: error.message
                })
            };
        }
        finally {
            if (browser !== null) {
                await browser.close();
            }
        }
    }
    else {
        response = {
            'statusCode': 400,
            'body': JSON.stringify({
                statusCode: 400,
                message: 'Bad Request'
            })
        };
    }

    return response;
};
```

## Usage of the API
Do a `POST` request to `http://127.0.0.1:3000/renderPdf` with the JSON Body 
```json
{
  "template": "<!DOCTYPE html>
<html>
<head>
	<title>Student's profile</title>
	<head>
	<meta charset=\"utf-8\">
</head>
<body>
	<div class=\"report-header\">
		<h3>title</h3>
		<p>date</p>
	</div>
	<div class=\"report-body\">
		<div class=\"info\">
			<p>Student: {{who}}</p>
		</div>
	</div>
	<div class=\"report-footer\">
		A generic Company - All Rights reserved.
	</div>\n</body>\n</html>",
  "data": {
    "who": "Borat"
  }
}
```
Using `sam build && sam local start-api` I'm not able to get a PDF with contents. Same if I deploy the function to AWS.

**Comment by baharev** [comment]
@katunch Please try to reduce your example; it has too many moving parts. In particular, try to **get rid of the following lines**:

```javascript
const fs = require('fs');
const path = require('path');
const Mustache = require('mustache');
```

and hard-code the generated HTML as a plain string in the JavaScript code. Make sure that the generated HTML is valid.

**Then get rid of everything specific to the Lambda** and write the PDF directly to disc by passing a `path: '/tmp/myexample.pdf',` to the ` await page.pdf({...})` call. We must be able to test everything without `sam`.

If that still doesn't reveal where the problem is, try eliminating as much of the HTML and the JavaScript code as possible. 

**Long story short: Please try to make your example minimal.**

**Comment by katunch** [comment]
hi @baharev thx for replying... I was able to solve the issue using `chrome-aws-lambda` as a [layer](https://github.com/shelfio/chrome-aws-lambda-layer) instead of a node module. Now it is working like a charm.
